pages/articles.tsx

```diff
@@ -11,7 +11,7 @@ interface IHomeProps {
 
 export default function Articles({ people }: IHomeProps) {
   return (
-    <Layout>
+    <Layout descriptionText={<></>}>
```

Comment:
Please return "descriptionText" to be optional and remove that change.

Reply:
Done